.env.example

```diff
@@ -1,6 +1,5 @@
 NODE_ENV=development
 PORT=4000
-APP_NAME=kaws-staging
```

Comment:
As far as I can tell, this was only used by the `ensureSSL` middleware, as far as I can tell.